cookbook/cookbook.ipynb

```diff
@@ -0,0 +1,2474 @@
+{
```

Comment:
Maybe call this `qcp_cookbook_v1` so that we can extend later?

Reply:
Does splitting the cookbook resolve this suggestion?

Reply:
Yes, lets merge.